src/displayBackend/wayland/Display.cpp

```diff
@@ -554,6 +568,12 @@ void Display::release()
 #endif
 #endif
 
+#ifdef WITH_WAYLAND_PRESENTATION_API
+	if(mCompositor)
```

Comment:
Use brackets for one line condition as well.

Reply:
Done